master/docs/tutorial/tour.rst

```diff
@@ -260,6 +260,13 @@ To use this you will need to install an additional package or two to your virtua
   pip install -U pip
   pip install cryptography pyasn1
 
+You will also need to generate an SSH host key for the Manhole server.
+
+.. code-block:: bash
+
+  mkdir -p /data/ssh_host_keys
+  ckeygen -t rsa -f /data/ssh_host_keys/ssh_host_rsa_key
```

Comment:
Ah, I see! I do not faced a lot with **host** keys before. I thinks it is easy to miss difference between client/host key, if user has not enough experience. Any way to clarify/accent it?